src/app/view_model_collection.cc

```diff
@@ -66,5 +71,15 @@ void ViewModelCollection::ExportContextPropertiesToQml( QQmlEngine* engine )
 void ViewModelCollection::SetRootObject( QObject* object )
 {
     m_eventFilter->FilterEventsDirectedAtThisObject( object );
+
+#if !defined( Q_OS_ANDROID )
+    if( m_opts->RunningGuiTests() )
```

Comment:
@mhoffman219 Unless we end up discovering some way in which the GuiTest(s) are fundamentally in conflict with how the PerformanceCounter works, then I think this is a good spot to place some sample code (since it won't execute other than during tests).

I ran the gui tests and I observed that this code is executing, but I still didn't manage to get to a point where the PerformanceCounter would dump out its statistics in `ReportRequest::ExportReport()`

Hoping you can help me add code here that would result in some printing of stats, even if the numbers are extremely boring when running the GuiTest of this app.

Reply:
Oh. And if you want to launch the app in GuiTest mode:
```
build/src/app/app -g
```

... and then additionally you might want to hack this `1` to make the app stay open longer so you can press buttons and cause more GUI activity before the app auto-closes. This line is where to hack:
- https://github.com/219-design/qt-qml-project-template-with-ci/blob/c37d79276a2d56/src/app/gui_tests.cc#L36